tests/gplates_1_3/screen-output

```diff
@@ -38,7 +31,7 @@ Number of degrees of freedom: 10,656 (6,528+864+3,264)
    Solving Stokes system... 30+16 iterations.
 
    Postprocessing:
-     RMS, max velocity:                  0.0256 m/year, 0.102 m/year
+     RMS, max velocity:                  0.0257 m/year, 0.0769 m/year
```

Comment:
The reason for the change in max velocity is probably that we now interpolate differently than before (in spherical coordinates instead of cartesian coordinates). If it is correct that this change is so big is rather difficult to verify. I discussed this issue with Rene and checked two points: 
1. In the visualization of the test outputs, the velocities at the surface point only in phi or theta direction. 
2. If I load the current day velocities as input in the test cases, the velocities at the surface seem to be in the usual range of present-day plate velocities. 
So far, we couldn't think of another way to find out more about that.